app/routers/monitoring.py

```
@@ -0,0 +1,451 @@
+    active_alerts = db.query(AlertInstance).filter(
+        AlertInstance.status == "active"
+    ).order_by(desc(AlertInstance.triggered_at)).all()
+    
+    return active_alerts
+
+
+@router.post("/sessions", response_model=dict)
+async def create_scrape_session(
+    search_term: str,
+    location: Optional[str] = None,
+    sites: List[str] = ["indeed", "linkedin", "glassdoor"],
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)